master/custom/builders.py

```diff
@@ -161,6 +161,7 @@
     # Linux armv7l (32-bit) GCC
     ("ARM Raspbian", "gps-raspbian", SlowNonDebugUnixBuild),
     ("ARM64 Raspbian", "stan-raspbian", SlowNonDebugUnixBuild),
+    ("ARM64 Raspbian", "savannah-raspbian", SlowNonDebugUnixBuild),
```

Comment:
These can’t be the same, they will conflict.

```suggestion
    ("ARMv8 Raspbian", "savannah-raspbian", SlowNonDebugUnixBuild),
```

How about this, per the [spec](https://en.m.wikipedia.org/wiki/ARM_Cortex-A76)?

Reply:
Or, since the Pi is more powerful, it could build debug?